CMakeLists.txt

```diff
@@ -196,6 +196,12 @@ IF (${ANY_INTERFACE_ENABLED})
 	ENDIF()
 
 	FIND_PACKAGE(SWIG ${SWIG_VERSION_MINIMUM} REQUIRED)
+
+	IF(SWIG_VERSION EQUAL 4.0.0)
+		MESSAGE("Swig 4.0.0 doesn't support directors with c++17")
```

Comment:
maybe we can add a "disabling" to the message?
Or, I would like an error even more as I don't like silent changes of settings....

Reply:
This error was only around in exactly 4.0.0?

Reply:
yes. it was introduced in 4.0.0, and will be fixed in 4.0.1.
do you mean something like
"Disabling swig directors: Not supported in swig 4.0.0"

Reply:
yes.
Or "Due to a bug, Swig directors are not supported with Swig 4.0.0. Please disable or upgrade Swig"

Reply:
ok, I changed it